external/Tocpp-Android-Template/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,5 +2,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
+zipStorePath=wrapper/dists0
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
```

Comment:
I still get an error that at least 7.5 is required, any ideas why? Maybe my Android Studio is too new?
```
FAILURE: Build failed with an exception.

* Where:
Build file '/home/mike/Projects/gfxreconstruct/angle_ext_dynamic_state_test/app/build.gradle' line: 1

* What went wrong:
A problem occurred evaluating project ':app'.
> Failed to apply plugin 'com.android.internal.version-check'.
   > Minimum supported Gradle version is 7.5. Current version is 7.3.3. If using the gradle wrapper, try editing the distributionUrl in /home/mike/Projects/gfxreconstruct/angle_ext_dynamic_state_test/gradle/wrapper/gradle-wrapper.properties to gradle-7.5-all.zip
```

Reply:
It doesn't work with 6.5 on my machine either. I always have to update this file to 7.5 to get my local `gfxrecon-tocpp` output to build.

Reply:
Some say this could be a cache problem.  When you rebuild, do you delete `android/.gradle` as well as all the generated `.cxx` and `.gradle` files from before?

Reply:
If this one line is changed to 7.5 does it work for you?   Or are there more changes involved?

Reply:
Pretty much yes. I need to do 2 things get the default output of tocpp to build:
- Change the version number here to 7.5
- Tweak the cmake include dirs so they find a more recent version of vulkan.h (due to my distro's libvulkan-dev package being pretty old at 1.3.204)

Reply:
An just to clarify, I have never had an issue building the capture layer or replay tool for android, just the output of tocpp.

Reply:
Updated to 7.5 for both "tocpp" and standard build.